WooCommerce/WooCommerceTests/ViewRelated/Shipping Label/WooShipping Create Shipping Labels/WooShippingAddCustomPackageViewModelTests.swift

```diff
@@ -0,0 +1,158 @@
+import XCTest
+@testable import WooCommerce
+import Yosemite
+
+final class WooShippingAddCustomPackageViewModelTests: XCTestCase {
+    func test_it_inits_with_empty_field_values() {
+        // Given/When
+        let viewModel = WooShippingAddCustomPackageViewModel()
+
+        // Then
+        XCTAssertNotNil(viewModel)
+        XCTAssertEqual(viewModel.fieldValues.isEmpty, true)
+        XCTAssertEqual(viewModel.packageType, WooShippingPackageType.box)
+        XCTAssertEqual(viewModel.showSaveTemplate, false)
+        XCTAssertEqual(viewModel.packageTemplateName, "")
+        XCTAssertEqual(viewModel.areFieldValuesInvalid, true)
+    }
+
+    func test_clear_field_values() {
+        // Given
+        let viewModel = WooShippingAddCustomPackageViewModel()
+
+        // When
+        viewModel.clearFieldValues()
```

Comment:
Given that it inits with empty field values, how about entering some values (e.g. the `viewModel.fillWithDummyFieldValues()` method you use below) before clearing them in this test?